### PR TITLE
Add info about VNC+Wayland compatibility (bsc#1202038)

### PR DESCRIPTION
**Pull request by ismaell**
### PR creator: Description

Warn users about Wayland incompatibility with VNC sessions.

### PR creator: Are there any relevant issues/feature requests?

* bsc#1202038

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [x] SLE 15 SP1
  - [x] SLE 15 SP0
- SLE 12
  - [x] SLE 12 SP5
  - [x] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by taroth21** [comment]
@ismaell : Many thanks for opening the PR and sorry for reviewing only now! I would have a few minor corrections for the PR (language and style issues). I could not push them from my local branch (based on your PR) back to your fork directly because I lack permissions. Do you want me to make code suggestions instead?

**Comment by ismaell** [comment]
Sure, suggestions would also help me improve.

File: xml/vnc.xml [hunk 1]
```diff
@@ -980,4 +980,22 @@ WINDOWMANAGER=icewm vncserver -geometry 1024x768</screen>
...
+  <title>Compatibility with Wayland</title>
+  <para>
+    The Remote Admininstration (VNC) feature relies on X11 and may result in an
+    empty screen if Wayland is enabled.
+
+    The display manager must be configured to use X11 instead of Wayland.
+
+    For <package>gdm</package>, add <literal>WaylandEnable=false</literal> under
+    the <literal>[daemon]</literal> section to the
+    <filename>/etc/gdm/custom.conf</filename> configuration file.
```
Comment (taroth21) [inline xml/vnc.xml:993]:
```suggestion
   For <package>gdm</package>, edit <filename>/etc/gdm/custom.conf</filename>.
   In the <literal>[daemon]</literal> section, add <literal>WaylandEnable=false</literal>
   to the configuration file.
```

File: xml/vnc.xml [hunk 2]
```diff
@@ -980,4 +980,22 @@ WINDOWMANAGER=icewm vncserver -geometry 1024x768</screen>
...
+
+    When logging in, the user must choose a X11-compatible session as well.
```
Comment (taroth21) [inline xml/vnc.xml:995]:
```suggestion
    When logging in, the user must choose an X11-compatible session as well.
```

File: xml/vnc.xml [hunk 3]
```diff
@@ -980,4 +980,22 @@ WINDOWMANAGER=icewm vncserver -geometry 1024x768</screen>
...
+
+    If you wish to remove the Wayland option for GNOME, you can remove and lock
```
Comment (taroth21) [inline xml/vnc.xml:997]:
```suggestion
    If you wish to remove the Wayland option for &gnome;, you can remove and lock
```

File: xml/vnc.xml (hunk 1 above)
Comment (taroth21) [reply xml/vnc.xml:993]: It is easier to follow the instructions when you first mention which file and which section to edit, and only then mention the parameter settings.

File: xml/vnc.xml [hunk 4]
```diff
@@ -980,4 +980,22 @@ WINDOWMANAGER=icewm vncserver -geometry 1024x768</screen>
    "Connection not encrypted!" message in that case).
   </para>
  </sect1>
+ <sect1 xml:id="sec-vnc-wayland">
+  <title>Compatibility with Wayland</title>
+  <para>
+    The Remote Admininstration (VNC) feature relies on X11 and may result in an
```
Comment (taroth21) [inline xml/vnc.xml:986]:
```suggestion
    The Remote Administration (VNC) feature relies on X11 and may result in an
```

**Review by taroth21 — CHANGES_REQUESTED** [review]
Hi, I have a few minor correction suggestions (language and style-related), otherwise I'm fine with the addition. Many thanks!

**Review by taroth21 — APPROVED** [review]

**Comment by taroth21** [comment]
Thank you, @ismaell! I have merged this to `main` now and will take care of backporting the changes to the other branches next week.

**Comment by taroth21** [comment]
@ismaell : All backports are done now, the changes will be published on documentation.suse.com with the next weekly SLE updates.

**Comment by ismaell** [comment]
Thanks.

**Pull request merged**